src/logger.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+const KafkaProducer = require('no-kafka');
+const configFunctions = require('../src/kafkaConfig');
+const kafkaConfig = configFunctions.getConfig();
+
+let producer;
+const initKafkaLoggingProducer = () => {
+  if (configFunctions.kafkaLogging) {
+    producer = new KafkaProducer.Producer({
+      connectionString: kafkaConfig.connectionString,
+      ssl: {
+        cert: kafkaConfig.sslCert,
+        key: kafkaConfig.sslKey,
+      },
+    });
+    return producer.init().catch((err) => {
+      throw new Error(`Failed to initialized Kafka producer for logging, error: ${err}`);
```

Comment:
use process.exit(1) since the intention is to NOT have the app running if it *should* use kafka but cannot initialize the producer